frontend/apps/reader/readerui.lua

```diff
@@ -756,9 +756,10 @@ function ReaderUI:notifyCloseDocument()
             self:closeDocument()
         else
             UIManager:show(ConfirmBox:new{
-                text = _("Do you want to save this document?"),
+                text = _("Do you want to save new highlights in this document to the pdf file?"),
```

Comment:
I would avoid mentioning PDF. That merely describes the current situation.

So I'd rather see something like: "(Do you want to) Save (this) document with highlights included?"

Reply:
We already mention pdf
https://github.com/koreader/koreader/blob/e79f161516764c7f0fa7f950275489f3c5c65149/frontend/ui/elements/common_settings_menu_table.lua#L587
Maybe use "annotation" to emphasize specific pdf feature?

Reply:
> That merely describes the current situation.

This is the situation we had for 10 years, and probably for the coming up 10 years :)
I think "annotation" would not help guessing it's about PDF.
This was already obscure to understand at the time, so I added "(write highlights into PDF)" - and I still think we need to see "PDF" to understand that stuff may be written in a PDF file.
So users don't go at expecting this can happen with EPUB.

Reply:
Can this be applied to djvu?

Reply:
No it's a pdf format feature.

Reply:
`(Do you want to) Write highlights into this PDF?`